spec/models/subject_spec.rb

```diff
@@ -102,7 +102,7 @@
       expect(expected.map(&:id)).to eq(lone_subject.ordered_locations.map(&:id))
     end
 
-    context "subject without location metadata", :focus do
```

Comment:
https://gist.github.com/zwolf/a2f20f7b673792bda95a156619d34acf

I haven't done it since.